source/publishing_data/02_harvesting_a_catalog/harvesters/datagouv.rst

```diff
@@ -17,19 +17,37 @@ Parameters
 
 .. list-table::
    :header-rows: 1
+   :widths: 25 50 25
 
    * * Name
      * Description
-     * Example
+     * Examples
    * * Organization ID (organization_id)
-     * The **id** of the organization you want to harvest (found here: https://www.data.gouv.fr/api/1/organizations/)
-     * 534fffa5a3a7292c64a7809e
-   * * Attachment (generate attachment)
-     * Check this if you want to attach resources as attachments instead of using them as sources
-     *
+     * The identifier of the organization you want to harvest. 
+ 
+       You can find the identifier for a specific organization as follows:
+
+       1. Go to https://www.data.gouv.fr/fr/organizations/.
+       2. Search for an organization.
+       3. Click on the desired result to open it.
+       4. On the desired organization page, you'll find the `ID` under the "Informations techniques" section.
+     * ``534fff75a3a7292c64a77de4`` is the identifier for etalab.
    * * Filter parameter name (filter_name)
-     * Name of the parameter you want to pass to the data.gouv.fr API to filter the request that will be performed
-     *
+     * The name of the parameter you want to pass to the data.gouv.fr API to filter the request that will be performed:
+       
+       - ``tag`` is used to filter datasets associated with a specific keyword.
+       - ``license`` is used to filter datasets published under a specific license.
+       - ``temporal_coverage`` is used to filter datasets with data covering a specific period.
+       - ``geozone`` is used to filter datasets with data covering a specific geographical area.
+       - ``granularity`` is used to filter datasets with a specific territorial coverage granularity.
+       - ``format`` is used to filter datasets published under a specific format.
+
+     * ``temporal-coverage``
    * * Filter parameter value (filter_value)
-     * Value of the above parameter
-     *
+     * The value of the filter parameter
+
+       The value for some filter parameters has a very specific syntax. You can perform a search on https://www.data.gouv.fr/fr/datasets/ to get some hints about possible values.
```

Comment:
can you add something like "When performing a search, the URL is of the form data.gouv.fr/fr/datasets/?[filter_name]=[filter_value], you can retrieve the values from the URL to set your harvester" to help users find the hints and build their harvesters? or is it too specific?

Reply:
Sure, it is way clearer. Thanks for the suggestion @cblsr!
Fixed in 0d98408